pyproject.toml

```diff
@@ -75,6 +75,11 @@ issues = "https://github.com/octoenergy/xocto/issues"
 [tool.setuptools]
 packages = ["xocto", "xocto.events", "xocto.storage"]
 
+[tool.setuptools.package-data]
+"xocto" = ["py.typed"]
+"xocto.events" = ["py.typed"]
+"xocto.storage" = ["py.typed"]
```

Comment:
TBH I'm not *certain* that we need these files for each of the packages listed. Mypy isn't clear on this.

Reply:
Would `[xocto] = ["**/py.typed"]` be more future-proof?